dotnetv3/AutoScaling/scenarios/AutoScale_Basics/AutoScale_BasicsTests/AutoScaleMethodsTests.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier:  Apache-2.0
+
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using AutoScale_Basics;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Amazon.AutoScaling;
+using Amazon.AutoScaling.Model;
+
+namespace AutoScale_Basics.Tests
+{
+    [TestClass()]
+    public class AutoScaleMethodsTests
+    {
+        private string _GroupName = "test-group-name";
+        private AmazonAutoScalingClient _Client = new AmazonAutoScalingClient();
+
+        // the Amazon Resource Name (ARN) of the IAM service linked role.
+        private readonly string _ServiceLinkedRoleArn = "<Enter Value>";
+
+        private readonly string _LaunchTemplateName = "AutoScaleLaunchTemplateTest";
+
+        [TestMethod()]
+        public async Task DeleteAutoScalingGroupTest()
+        {
+            var success = await AutoScaleMethods.DeleteAutoScalingGroupAsync(_Client, _GroupName);
+            Assert.IsTrue(success, "Could not delete the list.");
+        }
+
+        [TestMethod()]
+        public async Task DeleteAutoScalingGroupGroupNonexistentTest()
+        {
+            var success = await AutoScaleMethods.DeleteAutoScalingGroupAsync(_Client, "NonExistentSite");
+            Assert.IsFalse(success, "Could not delete the list.");
```

Comment:
I think the failure message is reversed here

Reply:
You're right. The case it is testing is that we're trying to delete a group that doesn't exist. Changed the message to "Could not find a group to delete."